src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1572,5 +1603,81 @@ public void TestWhere_NonStringCompare()
 {% endfor %}",
                 localVariables: Hash.FromAnonymousObject(new { products }));
         }
+
+        [Test]
+        public void TestConcat()
+        {
+            var array1 = new String[] { "one", "two" };
+            var array2 = new String[] { "alpha", "bravo" };
+
+            CollectionAssert.AreEqual(null, StandardFilters.Concat(null, null));
+            CollectionAssert.AreEqual(array1, StandardFilters.Concat(array1, null));
+            CollectionAssert.AreEqual(array1, StandardFilters.Concat(null, array1));
+            CollectionAssert.AreEqual(new[] { "one", "two", "alpha", "bravo" }, StandardFilters.Concat(array1, array2));
+            CollectionAssert.AreEqual(new[] { 1, 2, 3, 4 }, StandardFilters.Concat(new[] { 1, 2 }, new[] { 3, 4 }));
+        }
+
+        [Test]
+        public void TestConcat_LiquidSample_SingleFilter()
+        {
+            Helper.AssertTemplateResult(
+                expected: "\r\n\r\n\r\n\r\n- apples\r\n\r\n- oranges\r\n\r\n- peaches\r\n\r\n- carrots\r\n\r\n- turnips\r\n\r\n- potatoes\r\n",
+                template: @"{% assign fruits = 'apples, oranges, peaches' | split: ', ' %}
+{% assign vegetables = 'carrots, turnips, potatoes' | split: ', ' %}
+{% assign everything = fruits | concat: vegetables %}
+{% for item in everything %}
+- {{ item }}
+{% endfor %}");
+        }
+
+        [Test]
+        public void TestConcat_LiquidSample_ChainedFilters()
+        {
+            Helper.AssertTemplateResult(
+                expected: "\r\n\r\n\r\n\r\n\r\n- apples\r\n\r\n- oranges\r\n\r\n- peaches\r\n\r\n- carrots\r\n\r\n- turnips\r\n\r\n- potatoes\r\n\r\n- chairs\r\n\r\n- tables\r\n\r\n- shelves\r\n",
+                template: @"{% assign fruits = 'apples, oranges, peaches' | split: ', ' %}
+{% assign vegetables = 'carrots, turnips, potatoes' | split: ', ' %}
+{% assign furniture = 'chairs, tables, shelves' | split: ', ' %}
+{% assign everything = fruits | concat: vegetables | concat: furniture %}
+{% for item in everything %}
+- {{ item }}
+{% endfor %}");
+        }
+
+        [Test]
+        public void TestReverse()
+        {
+            var array = new String[] { "one", "two", "three" };
+            var arrayReversed = new String[] { "three", "two", "one" };
+
+            CollectionAssert.AreEqual(null, StandardFilters.Reverse(null));
+            CollectionAssert.AreEqual(arrayReversed, StandardFilters.Reverse(array));
+            CollectionAssert.AreEqual(array, StandardFilters.Reverse(arrayReversed));
+            CollectionAssert.AreEqual(new[] { 3, 2, 2, 1 }, StandardFilters.Reverse(new[] { 1, 2, 2, 3 }));
+        }
+
+        /// <summary>
+        /// Reverses the order of the items in an array. reverse cannot reverse a string.

```

Comment:
Should you have a test case for calling reverse on the string? While this shouldn't work, it is important that we have explicit test as some users will attempt that before they discover they need to first split, then reverse, then join.

Reply:
Added test case.